ProjectName/Sources/Presentation/Navigator/Navigator.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  Navigator.swift
+//
+
+import UIKit
+
+// sourcery: AutoMockable
+protocol Navigatable: AnyObject {
+
+    func show(scene: Navigator.Scene, sender: UIViewController?, transition: Navigator.Transition)
+    func dismiss(sender: UIViewController?)
+    func pop(sender: UIViewController?)
+    func popToRoot(sender: UIViewController?)
+}
+
+final class Navigator {
+
+    private func viewController(from scene: Scene) -> UIViewController {
+        // Make view controller
```

Comment:
We should make the project "buildable" after generating the project, so I suggest:
```suggestion
        #warning("Create view controllers here")
        return HomeViewController()
```